apps/src/fish/FishView.jsx

```diff
@@ -53,23 +47,98 @@ class FishView extends React.Component {
     mobilePortraitWidth: PropTypes.number.isRequired
   };
 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      windowWidth: $(window).width(),
+      windowHeight: $(window).height(),
```

Comment:
Won't this still run into issues with android? i.e. with width always being the shortest measurement of the phone and height always being the longest measurement on the phone?

Reply:
It looks like it worked okay, but probably because we only see the results of this code in landscape orientation, in which case `window.innerHeight`, which can be the screen's width on an Android device in such a case, is going to be bigger than the space allocated for the app in the form of the height of the `.codeApp` div, and so the Oceans app won't be reduced in size.

That said, I've updated to cover the Android case anyway.